schema/dataKeys.json

```diff
@@ -437,6 +437,50 @@
                   }
                 }
               },
+              "productInfo": {
+                "type": "object",
+                "properties": {
+                  "productName": {
+                    "type": "string",
+                    "description": "productName in content gateway"
+                  },
+                  "productCode": {
+                    "type": "string",
+                    "description": "productCode in content gateway"
+                  },
+                  "productVersionName": {
+                    "type": "string",
+                    "description": "productVersionName in content gateway"
+                  }
+              },
```

Comment:
The json is malformed. It seems the closing `}` is missing for `"properties"` here. Not sure if anything else.

Reply:
I'm going to fix it, just a sec